lib/mixlib/install/cli.rb

```diff
@@ -49,6 +49,7 @@ def list_versions(product_name, channel)
         desc: "Print artifact attributes",
         type: :boolean
       def download(product_name)
+        say_status("Warn", "Product name `delivery` will be deprecated in a future release. Please use `automate`") if product_name == "delivery"
```

Comment:
I would use the terminology `Product key` so we are consistent in what we call those things.